templates/projects/external_config.html

```diff
@@ -23,14 +23,24 @@ <h3>{{_('No external services have been configured')}}</h3>
 <p>For more information on these settings, please consult the <a href={{config.EXT_CONFIG_DOCS}}>External Configurations documentation</a>.</p>
 {% endif %}
 
-<h3>Authorized Services</h3>
-<span>{{ authorized_services|join(', ') }}</span>
 
 {% for (form_name, display, form) in forms %}
     <h3>{{display}}</h3>
     {{render_form(form, form_id=form_name, action_url=url_for('project.ext_config', short_name=project.short_name), action_text='Submit', class_='', btn_name=form_name, btn_class='btn btn-default', btn_value=None)}}
 {% endfor %}
 
+<h3>Authorized Services</h3>
+{% if authorized_services is defined and  authorized_services|length > 0 %}
```

Comment:
Can this be optimized to: 
```python
{% if authorized_services %}
```

Reply:
From what I know about Jinja templating language, no: https://www.shellhacks.com/jinja2-check-if-variable-empty-exists-defined-true/

Reply:
Feel free to test the optimized version. Otherwise, either would be fine. Thank you.